Sming/Makefile-windows.mk

```diff
@@ -9,7 +9,7 @@ SDK_BASE	?= $(ESP_HOME)/ESP8266_SDK
 SDK_TOOLS	 ?= $(ESP_HOME)/utils
 
 # Other tools mappings
-ESPTOOL		 ?= $(SDK_TOOLS)/esptool.exe
+ESPTOOL		 ?= $(SDK_TOOLS)/ESP8266/esptool.exe
```

Comment:
> Maybe then remove this addition and also the one on memanalyzer and define it by 

@ADiea That would be great. Can you please create a PR with those changes?

Another question: Do you know how we can use the latest version of Sming in `chocolate`?